documents/refund.md

```diff
@@ -96,8 +103,12 @@ Razorpay::QrCode.create(paymentId,{
 
 ### Fetch multiple refunds for a payment
 
-```js
-instance.payments.fetch_multiple_refund(paymentId,option)
+```rb
+paymentId = "pay_FIKOnlyii5QGNx"
+
+option = {"count":1}
+
+Razorpay::Payment.fetch_multiple_refund(paymentId,option)
```

Comment:
Where is this declared?

Reply:
its declared in payment module 

https://github.com/razorpay/razorpay-ruby/blob/payment_update/lib/razorpay/payment.rb#L42

Reply:
Part of separate branch. Pls describe this function in the correct place.

Reply:
according to [doc](https://razorpay.com/docs/api/refunds/#fetch-multiple-refunds-for-a-payment) this function is belongs to refund module that why i have added this function in refund.md